dev/docker/iceberg-rest-server/rewrite_config.py

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env 
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import os
+
+env_map = {
+  "GRAVITINO_IO_IMPL" : "io-impl",
+  "GRAVITINO_URI" : "uri",
+  "GRAVITINO_WAREHOUSE" : "warehouse",
+  "GRAVITINO_CREDENTIAL_PROVIDER_TYPE" : "credential-provider-type",
+  "GRAVITINO_GCS_CREDENTIAL_FILE_PATH" : "gcs-credential-file-path",
+  "GRAVITINO_S3_ACCESS_KEY" : "s3-access-key-id",
+  "GRAVITINO_S3_SECRET_KEY" : "s3-secret-access-key",
+  "GRAVITINO_S3_REGION" : "s3-region",
+  "GRAVITINO_S3_ROLE_ARN" : "s3-role-arn",
+  "GRAVITINO_S3_EXTERNAL_ID" : "s3-external-id"
+}
+
+init_config = {
+  "catalog-backend" : "jdbc",
+  "jdbc-driver" : "org.sqlite.JDBC",
+  "uri" : "jdbc:sqlite::memory:",
+  "jdbc-user" : "iceberg",
+  "jdbc-password" : "iceberg",
+  "jdbc-initialize" : "true",
+  "jdbc.schema-version" : "V1"
+}
+
+
+def parse_config_file(file_path):  
+    config_map = {}  
+    with open(file_path, 'r') as file:  
+        for line in file:  
+            stripped_line = line.strip()  
+            if stripped_line and not stripped_line.startswith('#'):  
+                key, value = stripped_line.split('=')  
+                key = key.strip()  
+                value = value.strip()  
+                config_map[key] = value  
+    return config_map  
+
+config_prefix = "gravitino.iceberg-rest."
+
+def update_config(config, key, value):
+    config[config_prefix + key] = value
+  
+config_file_path = 'conf/gravitino-iceberg-rest-server.conf'
+config_map = parse_config_file(config_file_path)
+
+for k, v in init_config.items():
+    update_config(config_map, k, v)
```

Comment:
You can make the dict `config_map` as the method parameter and do not need to call the method several time.

Reply:
it could improve the performance, but the current implementation is more extensiable, I think this is more important, WDYT?

Reply:
What do you mean by `more extensiable`? do you mean we can easily support more properties in this way?

Reply:
yes, like what `env_map` is processed

Reply:
Then how to process `env_map`?

Reply:
Since the method contains only one simple sentence, I can't find the difference that you replace it with ` config[config_prefix + key] = value` directly if insist on this the way. 

```
def update_config(config, key, value):
    config[config_prefix + key] = value
```

Reply:
If just for performance, I prefer not to change it. You suggest to pass map to `update_config`, then how to process `env_map` then?